_posts/2018-07-31-how-to-move-to-https.md

```diff
@@ -0,0 +1,100 @@
+
+We have recently helped a client who has been publishing articles at a high cadence for over 10 years with many different stakeholders over that period. Practices weren't consistent and uncovering all the ways in which HTTP resources were served from the page was a substantial undertaking. Therefore, be prepared for a time investment here, there may be many areas to audit to ensure all assets from all your pages are being served over HTTPS. Some common ways to experience mixed content in which HTTP assets are served from a site whose HTML is served over HTTPS are:
+
+- Hard-coding a resource: e.g. `http://www.example.com/img/insecure-image.jpg`
+- Using a 3rd-party library or ad network reference: `http://www.example.com/js/analytics.js`
+  - This is common for libraries that haven't been updated in a while. Most all of them now serve the same assets over the same path securely with HTTPS.
+
+Even [practices that were previously encouraged by Paul Irish](https://www.paulirish.com/2010/the-protocol-relative-url/), a leading web architect for the Google Chrome team may have contributed to your mixed content problem, so don't feel bad. But, there will likely be work to be done.
+
+#### The risk of mixed content
+
+These "not secure" bits of mixed-content [expose the same risk](https://developers.google.com/web/fundamentals/security/prevent-mixed-content/what-is-mixed-content#mixed_content_weakens_https
+) that your HTML does when served over HTTP, so browsers rightfully show the browser that the experience on your site is "not secure".
+
+Mixed content is categorized in two ways: active and passive. An asset that can contribute to passive mixed content would be an image; it doesn't interact with the page but is merely presented on the page. An example that can be an active asset of mixed content is a javascript file or stylesheet since its purpose is to manipulate the page. Passive mixed content, albeit categorically less severe than active, still exposes risk to an attacker to learn a lot about an individual's browsing patterns and even trick them into taking different actions than they intend to on the page. Therefore passive mixed content still constitutes enough of a threat for the browser to issue a warning display.
+
```

Comment:
"...still exposes risk to an attacker ~to~ who can learn a lot about an individual's browsing patterns...."